src/templates/en/2019/index.html

```diff
@@ -105,8 +105,8 @@ <h2>
         Start exploring
       </a>
     </div>
-    <img class="intro-image-2019" src="/static/images/home-hero-2019.png" alt="The Web Almanac 2019 edition">
-    <img class="intro-image" src="/static/images/home-hero.png" alt="The Web Almanac 2019 edition">
+    <img loading="lazy" class="intro-image-2019" src="/static/images/home-hero-2019.png" alt="The Web Almanac 2019 edition">
```

Comment:
Wouldn't bother lazy loading these - though we should do better here to only load once. See https://github.com/HTTPArchive/almanac.httparchive.org/issues/361

```suggestion
    <img class="intro-image-2019" src="/static/images/home-hero-2019.png" alt="The Web Almanac 2019 edition">
```